Changelog.md

```diff
@@ -17,6 +17,7 @@
 ### Misc Changes
 
 - [#468]: Content of `DeError::Unsupported` changed from `&'static str` to `Cow<'static, str>`
+- [#468]: Ensure, that map keys could only types that serialized as primitives only
```

Comment:
Ensure that map keys are restricted to types that can be serialized as primitives